packages/config/src/layer2s/arbitrum.ts

```diff
@@ -131,6 +131,8 @@ export const arbitrum: Layer2 = {
         },
       ],
     },
+    tvlTooltip:
+      'TVL includes canonically bridged assets, natively minted assets and externally bridged assets.',
```

Comment:
TVL includes canonically bridged assets, native ARB and USDC directly minted on Arbitrum